server/game/cards/01_SOR/units/RuggedSurvivors.ts

```diff
@@ -0,0 +1,28 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardLocation } from '../../../core/Constants';
+
+export default class RuggedSurvivors extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '4599464590',
+            internalName: 'rugged-survivors'
+        };
+    }
+
+    public override setupCardAbilities () {
+        // valball : use addTriggeredAbility because addOnAttackAbility does not reference immediateEffect
+        this.addTriggeredAbility({
+            title: 'You may draw a card if you control a leader unit',
```

Comment:
We do not include the "you may" in the ability title because it gets added to the button, and it is a little weird to have a button that says "You may do [X]". So in this case should just be "Draw a card if you control a leader unit". Ditto for other card titles in here that say "you may"